src/components/shared/basic/SearchResultItem/index.tsx

```diff
@@ -106,6 +109,25 @@ export const SearchResultItem = ({
   const { t } = useTranslation()
   const dispatch = useDispatch()
   const navigate = useNavigate()
+  const [fetchDocumentById] = useFetchDocumentByIdMutation()
+  const [file, setFile] = useState<string>()
+
+  const fetchImage = async (appId: string, documentId: string) => {
+    try {
+      const response = await fetchDocumentById({ appId, documentId }).unwrap()
+      const file = response.data
+      setFile(URL.createObjectURL(file))
+    } catch (error) {
+      console.error(error, 'ERROR WHILE FETCHING IMAGE')
+    }
+  }
+
+  useEffect(() => {
+    if (item.leadPictureId) {
+      fetchImage(item.id, item.leadPictureId)
+    }
+  }, [item])
+
```

Comment:
delete all this added lines - the image component from our shared components is handling this